packages/suite/src/middlewares/wallet/discoveryMiddleware.ts

```diff
@@ -45,7 +45,8 @@ export const prepareDiscoveryMiddleware = createMiddlewareWithExtraDeps(
         if (action.type === ROUTER.LOCATION_CHANGE) {
             interruptionIntent =
                 getApp(action.payload.url) !== 'wallet' &&
-                getApp(action.payload.url) !== 'dashboard';
+                getApp(action.payload.url) !== 'dashboard' &&
+                getApp(action.payload.url) !== 'settings';
```

Comment:
Lets discuss this on Monday, this is the topic I want to raise if we really want this.